docs/private-dns/overview.md

```diff
@@ -175,8 +175,17 @@ In the Advanced section, there are three options that can be customized:
 
 Here you can manage an access to your DNS server by configuring the following settings:
 
-- Allowed clients. Specify which clients are permitted to use your DNS server
+- Allowed clients. Specify which clients are permitted to use your DNS server. Please that allowed clients are not counted in added access rules, only disallowed clients and domains
```

Comment:
(Please) note that allowed clients are not counted in added access rules, only disallowed clients and domains